pkg/upgrade/upgrade.go

```diff
@@ -21,11 +23,33 @@ func ManagedInstances(ctx context.Context, c client.Client, reader client.Reader
 
 	list := &v1.JaegerList{}
 	identity := viper.GetString(v1.ConfigIdentity)
-	opts := client.MatchingLabels(map[string]string{
+	opts := []client.ListOption{}
+	opts = append(opts, client.MatchingLabels(map[string]string{
 		v1.LabelOperatedBy: identity,
-	})
-	if err := reader.List(ctx, list, opts); err != nil {
-		return tracing.HandleError(err, span)
+	}))
+
+	// if set and np cluster scope permission, skip
+	// if not set, treat as true
+	if (viper.IsSet("has-cluster-permission") && viper.GetBool("has-cluster-permission")) || !viper.IsSet("has-cluster-permission") {
+		if err := reader.List(ctx, list, opts...); err != nil {
+			if strings.HasSuffix(err.Error(), "cluster scope") {
+				log.WithError(err).Warn("List indentity failed with cluster scope")
```

Comment:
`Listing of cluster-wide instances belonging to this operator failed.` is probably more descriptive.